srcpkgs/PopCorn/files/popcorn/log/run

```diff
@@ -0,0 +1 @@
+/usr/bin/vlogger
```

Comment:
Shouldn't this have a #! and an exec?

Reply:
The file is a symlink to /usr/bin/vlogger.